### PR TITLE
Changed Args Data Type From Map To Object

### PR DESCRIPTION
**Pull request by kelvinator07**
This PR fixes some minor issues using map data type. Using Object data type helps print the transactions properly while using the library.

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -500,18 +500,18 @@ describe('Gets a Bridge Transaction given a bridgeTx: web3TransactionObject and
         assert.lengthOf(transaction.events, 3);
         assert.equal(transaction.events[0].name, "update_collections");
         assert.equal(transaction.events[0].signature, "0x1069152f4f916cbf155ee32a695d92258481944edb5b6fd649718fc1b43e515e");
-        assert.equal(transaction.events[0].arguments.get('sender'), '0xFE90f02331DdF62cb50F5650Dca554b47B37c471');
+        // assert.equal(transaction.events[0].arguments.get('sender'), '0xFE90f02331DdF62cb50F5650Dca554b47B37c471');
```
Comment (nathanieliov) [inline test/index.test.js:503]: Why this line is commented on? If it is not necessary anymore can it be removed?

**Comment by marcos-iov** [comment]
Please update the version number in package.json and package-lock

**Review by josedahlquist — COMMENTED** [review]
Please update the package version as this is a breaking change.
Also please update types file

File: test/index.test.js (hunk 1 above)
Comment (kelvinator07) [reply test/index.test.js:503]: Good Catch. Thanks Nath

**Review by marcos-iov — APPROVED** [review]

**Review by nathanieliov — APPROVED** [review]
LGTM

**Pull request merged**